addons/crm/static/src/activity_menu_patch.js

```diff
@@ -22,7 +22,7 @@ patch(ActivityMenu.prototype, {
         const context = {};
         if (group.model === "crm.lead") {
             this.dropdown.close();
-            if (filter === "my") {
+            if (filter === "my" || filter === "all") {
                 context["search_default_activities_overdue"] = 1;
                 context["search_default_activities_today"] = 1;
             } else {
```

Comment:
While we are here, can't we have the same code as in mail and have somethign more explicit (string concatenation is always a bad idea as when you grep something you don't kind it)
```
        } else if (filter === "overdue") {
            context["search_default_activities_overdue"] = 1;
        } else if (filter === "today") {
            context["search_default_activities_today"] = 1;
        } else if (filter === "upcoming_all") {
            context["search_default_activities_upcoming_all"] = 1;
        }
```